grpc-server/src/main/java/com/netflix/conductor/grpc/server/service/HealthServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.netflix.conductor.grpc.server.service;
+
+import com.netflix.runtime.health.api.HealthCheckAggregator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+
+import io.grpc.health.v1.HealthCheckRequest;
+import io.grpc.health.v1.HealthCheckResponse;
+import io.grpc.health.v1.HealthGrpc;
+import io.grpc.stub.StreamObserver;
+
+public class HealthServiceImpl extends HealthGrpc.HealthImplBase {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HealthServiceImpl.class);
+    private static final GRPCHelper GRPC_HELPER = new GRPCHelper(LOGGER);
+
+    private final HealthCheckAggregator healthCheck;
+
+    @Inject
+    public HealthServiceImpl(HealthCheckAggregator healthCheck) {
```

Comment:
HealthModule needs to be installed.

Reply:
That is done in the `ServerModule`, which I believe at the moment is the correct place.

https://github.com/Netflix/conductor/pull/698/files#diff-d0cb0cb5585c979ec521f16dd0acce59R43